test/e2e/utils/labels.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import "strings"
+
+const (
```

Comment:
is there a way to import these annotations from the `api` library instead of hardcoding them here?

Reply:
I'm switching over to importing the annotation labels